### PR TITLE
create readme

### PR DESCRIPTION
**Pull request by begemot1987**

### CODE REVIEW

**Review by TankevychO — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+   * Maven
+   * HTML, CSS
+   * JSTL
+## Setup recommendations
+   1. Install MySQL
+   2. Configure Apache Tomcat
+   3. Create all necessary tables using script from `init_db.sql` file.
+      **WARNING! This script will drop `taxi` schema if you have it. So be careful**
+   4. Clone this project into your IDE
+   5. Replace `USERNAME` and `PASSWORD` values in `src/main/java/mate/util/ConnectionUtil.java` with your data
```
Comment (Nick97-git) [inline README.md]: what about URL?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+   6. Change `src/main/resources/log4j2.xml`. At line `<File name="LogToFile" fileName="E:\Mate\Projects\taxi_service\logs\app.log">` 
+replace fileName with absolute path to `.log` file
```
Comment (Nick97-git) [inline README.md]:
```suggestion
   6. Change `src/main/resources/log4j2.xml`. Replace fileName with absolute path to your log file at line `<File name="LogToFile" fileName="E:\Mate\Projects\taxi_service\logs\app.log">` 
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+   * JDBC
+   * JSP
+   * Maven
+   * HTML, CSS
+   * JSTL
+## Setup recommendations
+   1. Install MySQL
+   2. Configure Apache Tomcat
+   3. Create all necessary tables using script from `init_db.sql` file.
+      **WARNING! This script will drop `taxi` schema if you have it. So be careful**
```
Comment (Nick97-git) [inline README.md]: move warning to the next line

**Review by Nick97-git — CHANGES_REQUESTED** [review]
- remove redundant `/` from urls like /drivers/, /cars/, /manufacturers/
- at the moment your path looks like src/main/java/mate... We should rename "mate" to something else, like "taxi". After you do that don't forget to change the value you pass to the Injector.getInstance() everywhere you use it
- DeleteCarController, DeleteDriverController, DeleteManufacturerController -> remove /all from /cars/all, /drivers/all, /manufacturers/all url patterns
- ConnectionUtil -> add default values for USERNAME, PASSWORD, URL, like "YOUR_URL" for example

**Review by Nick97-git — APPROVED** [review]

**Pull request merged**